src/models/forecasts.jl

```diff
@@ -3,11 +3,12 @@ abstract type
 end
 
 struct Deterministic <: Forecast
+    device::PowerSystemDevice
```

Comment:
@claytonpbarrows  should we add a field that specifies the forecast use? For instance Day Ahead vs Real Time?

Reply:
I don't think so. I think that should be driven by some combination of the existing fields (which aren't completely clear to me). Basically we need to know:
 - What the forecast is for (device)
 - When the forecast is available (initial time?)
 - The horizon of the forecast (e.g. the timestamps for the values in the forecast timeseries). (interval, resolution, and horizon?)